internal/tasks/manifests.go

```diff
@@ -748,8 +748,12 @@ func (j *Janitor) doSecurityCheck(ctx context.Context, securityInfo *models.Triv
 
 	securityInfo.VulnerabilityStatus = newVulnerabilityStatus
 
-	// regular recheck loop (vulnerability status might change if Trivy adds new vulnerabilities to its DB)
-	securityInfo.NextCheckAt = j.timeNow().Add(j.addJitter(1 * time.Hour))
+	if newVulnerabilityStatus == models.RottenVulnerabilityStatus {
+		securityInfo.NextCheckAt = None[time.Time]()
```

Comment:
Please add a comment explaining why we're doing this. Same on EnrichReport. One of them can point at the other to avoid duplication, but we should be explaining why we're treating rotten images in this specific way.